fastjet/internal/delaunay/delaunay.go

```diff
@@ -162,6 +169,36 @@ func (d *Delaunay) insertInside(p *Point, t *Triangle) []*Point {
 // insertOnEdge inserts a point on an Edge between two triangles. It returns the points
 // whose nearest neighbor changed.
 func (d *Delaunay) insertOnEdge(p *Point, t *Triangle) []*Point {
+	// Check if p is a duplicate
+	switch {
```

Comment:
can `t` be made of 2 or even 3 duplicates?
e.g.: can `p.Equals(t.A)` and `p.Equals(t.B)` be both true?
presumably a `Triangle` can't be made like so, but... as we now have some degeneracy...

Reply:
No, because NewTriangle panics when three points are colinear

Reply:
good :)

Reply:
Should I change it to let triangles be composed of duplicates?

Reply:
no, I believe it's fine as is.
we'll cross that bridge when/if we need to.